src/main/java/de/qabel/desktop/ui/accounting/AccountingController.java

```diff
@@ -82,4 +89,66 @@ protected void addIdentityWithAlias(String alias) {
 			clientConfiguration.selectIdentity(identity);
 		}
 	}
+
+	@FXML
+	protected void handleImportIdentityButtonAction(ActionEvent event) {
+
+		FileChooser chooser = new FileChooser();
+		chooser.setTitle("Choose Download Folder");
+		File file = chooser.showOpenDialog(identityList.getScene().getWindow());
+		try {
+			String content = readFile(file);
+			Identity i = gson.fromJson(content, Identity.class);
+			identityRepository.save(i);
+		} catch (IOException | PersistenceException e) {
+			e.printStackTrace();
+		}
+	}
+
+	@FXML
+	protected void handleExportIdentityButtonAction(ActionEvent event)  {
+		DirectoryChooser chooser = new DirectoryChooser();
+		chooser.setTitle("Choose Download Folder");
+		File dir = chooser.showDialog(identityList.getScene().getWindow());
+		Identity i = clientConfiguration.getSelectedIdentity();
+		String json = gson.toJson(i);
+		try {
+			saveFile(json,i.getAlias(), dir);
+			loadIdentities();
+		} catch (IOException | QblStorageException e) {
+			e.printStackTrace();
+		}
+	}
+
+	void saveFile(String json,String name, File dir) throws IOException, QblStorageException {
+
+		InputStream stream = new ByteArrayInputStream(json.getBytes(StandardCharsets.UTF_8));
+		byte[] buffer = new byte[stream.available()];
+		stream.read(buffer);
+
+		File targetFile = new File(dir.getPath() + "/" + name + ".json");
+		OutputStream outStream = new FileOutputStream(targetFile);
+		outStream.write(buffer);
+	}
+
+
+
+	String readFile(File f) throws IOException {
+		FileReader fileReader = new FileReader(f);
+		BufferedReader br = new BufferedReader(fileReader);
+
+		try {
+			StringBuilder sb = new StringBuilder();
+			String line = br.readLine();
+
+			while (line != null) {
+				sb.append(line);
+				sb.append("\n");
```

Comment:
this line is causing a linebreak more at the end. Only add a \n if another line is following..